create_derivatives/routines.py

```diff
@@ -299,20 +277,20 @@ def create_manifest(self, identifier, obj_data):
             obj_data (dict): Data about the archival object.
         """
         manifest_path = Path(self.manifest_dir, "{}.json".format(identifier))
-        page_number = 1
         manifest = self.fac.manifest(ident="{}{}".format(self.fac.prezi_base, identifier), label=obj_data["title"])
         manifest.set_metadata({"Date": obj_data["dates"]})
         manifest.thumbnail = self.set_thumbnail(self.jp2_files[0].stem)
         sequence = manifest.sequence(ident=identifier)
         for file in self.jp2_files:
+            page_number = int(get_page_number(file))
```

Comment:
I'm wondering if there's a way to avoid calling zfill(4) twice in two different places - it feels like it would be really easy for someone to change it one place and not the other. When you call `get_page_number` here, before you coerce to an integer, do you get the zero-padded string? If so, I might suggest rethinking this logic so that you use that zero-padded string in line 293, and then coerce to an integer in line 295...